spec/rails_helper.rb

```diff
@@ -23,7 +23,7 @@
 # directory. Alternatively, in the individual `*_spec.rb` files, manually
 # require only the support files necessary.
 #
-Dir[Rails.root.join("spec", "support", "**", "*.rb")].each { |f| require f }
+Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
```

Comment:
rubocop has changed it's mind about separators because `/` is normalised by Ruby to be whatever the separator on the platform is (e.g. it becomes `\` on Windows

Reply:
:+1: TIL